docs/HACKING.md

```diff
@@ -160,11 +160,17 @@ commands like this:
 ```
 python infra/helper.py build_image systemd
 python infra/helper.py build_fuzzers --sanitizer memory systemd ../systemd
+python infra/helper.py check_build --sanitizer memory -e ALLOWED_BROKEN_TARGETS_PERCENTAGE=0 systemd
```

Comment:
I'm not sure why it's "memory" here. address and undefined along with `--engine=[afl,honggfuzz,libfuzzer]` should be added here as well I think

Reply:
It should be replaced with something like
```
# ./infra/helper.py pull_images
# ./infra/helper.py build_image --no-pull elfutils
# for sanitizer in address undefined memory; do
#   for engine in libfuzzer afl honggfuzz; do
#     ./infra/helper.py build_fuzzers --clean --sanitizer=$sanitizer --engine=$engine elfutils PATH/TO/ELFUTILS
#     ./infra/helper.py check_build --sanitizer=$sanitizer --engine=$engine -e ALLOWED_BROKEN_TARGETS_PERCENTAGE=0 elfutils
#   done
# done
#
# ./infra/helper.py build_fuzzers --clean --architecture=i386 elfutils PATH/TO/ELFUTILS
# ./infra/helper.py check_build --architecture=i386 -e ALLOWED_BROKEN_TARGETS_PERCENTAGE=0 elfutils
#
# ./infra/helper.py build_fuzzers --clean --sanitizer=coverage elfutils PATH/TO/ELFUTILS
# ./infra/helper.py coverage --no-corpus-download --fuzz-target=fuzz-dwfl-core --corpus-dir=PATH/TO/ELFUTILS/tests/fuzz-dwfl-core-crashes/ elfutils
```